### PR TITLE
Drive income (other sources) pages

### PR DESCRIPTION
**Pull request by jessieay**
* Check boxes related to sources of "other" income
* Fill in amounts for each source
* Open Question: what should we check for people who have social security
income" and/or "foster care / adoption" income?

screen shot 2017-09-25 at 11 55 53 am

### CODE REVIEW

**Comment by jessieay** [comment]
[Other types of income in MI Bridges](https://trello.com/c/FE7Ia91i/338-other-types-of-income-in-mi-bridges)

**Comment by jessieay** [comment]
[Money other sources follow up pages in MI Bridges](https://trello.com/c/VFNU1Hcn/337-money-other-sources-follow-up-pages-in-mi-bridges)

**Comment by jessieay** [comment]
@jayroh im gonna ignore this CC warnings

File: lib/mi_bridges/driver/others_types_of_income_page.rb [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+module MiBridges
+  class Driver
+    class OtherTypesofIncomePage < BasePage
+      TITLE = "Other Types of Income"
+
+      def setup; end
+
+      def fill_in_required_fields
+        # social security?
+        # foster care / adoption
```
Comment (adarsh) [inline lib/mi_bridges/driver/others_types_of_income_page.rb]: Style: code comments? Leftover todos?

File: lib/mi_bridges/driver/others_types_of_income_page.rb [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      def check_other_income
+        check_in_section(
+          section_name,
+          condition: additional_income?("other"),
+          for_label: "Other Income",
+        )
+      end
+
+      def section_name
+        "star#{first_name_section(primary_member).capitalize}sIncomeInformation"
```
Comment (adarsh) [inline lib/mi_bridges/driver/others_types_of_income_page.rb]: Maybe consider extracting something like:

```
def pluralized_first_name_of_primary_member
  first_name_section(primary_member).capitalize
end
```
?

Seems like a lot of activity for an interpolated string 🎞

File: lib/mi_bridges/driver/others_types_of_income_page.rb (hunk 1 above)
Comment (jessieay) [reply lib/mi_bridges/driver/others_types_of_income_page.rb]: see PR commit msg for context on this. Think we need to page @alanjosephwilliams for a final verdict

**Pull request merged**